Habitica/src/main/java/com/habitrpg/android/habitica/ui/adapter/inventory/StableRecyclerAdapter.kt

```diff
@@ -118,25 +123,49 @@ class StableRecyclerAdapter : androidx.recyclerview.widget.RecyclerView.Adapter<
             }
             ownedTextView.visibility = View.VISIBLE
             this.imageView.alpha = 1.0f
+            this.titleView.alpha = 1.0f
+            this.ownedTextView.alpha = 1.0f
+
             val imageName = if (itemType == "pets") {
-                "Pet-" + item.key
+                "Pet_Egg_" + item.animal
             } else {
                 "Mount_Icon_" + item.key
             }
-            this.ownedTextView.text = animal?.numberOwned?.toString()
-            ownedTextView.visibility = if (animal?.numberOwned == 0 || animal?.type == "special") View.GONE else View.VISIBLE
-            imageView.background = null
-            DataBindingUtils.loadImage(imageName) {
-                val drawable = BitmapDrawable(context?.resources, if (item.numberOwned > 0) it else it.extractAlpha())
-                Observable.just(drawable)
-                        .observeOn(AndroidSchedulers.mainThread())
-                        .subscribe(Consumer {
-                            imageView.background = drawable
-                        }, RxErrorHandler.handleEmptyError())
-            }
-            if (item.numberOwned <= 0) {
-                this.imageView.alpha = 0.1f
+
+            context?.let {
+
+                var owned = item.numberOwned?.toString()
+                var totalNum = item.totalNumber?.toString()
+
+
+                this.ownedTextView.text = context?.getString(R.string.pet_ownership_fraction, owned, totalNum)

```

Comment:
If you change the string from `%1$s/%2$s` to `%1$d/%2$d` you should be able to pass in the integers directly without using the toString() method here.